packages/smooth_app/lib/helpers/product_compatibility_helper.dart

```diff
@@ -71,6 +71,8 @@ class ProductCompatibilityHelper {
   String? getFormattedScore() {
     if (_score == null || status == MatchedProductStatusV2.UNKNOWN_MATCH) {
       return null;
+    } else if (_score == 0) {
+      return '0';
```

Comment:
What about 1? Are you OK with displaying 01?

Reply:
Honestly, this PR is only for fan service.
We're not consistent in changing it